src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -563,6 +565,24 @@ void ApplyLiteralProperties()
 
 				writer.AppendLineIndented(";");
 			}
+
+			void AttachUnhandledExceptionHandler()
+			{
+				writer.AppendLineIndented($"#if DEBUG");
```

Comment:
Usually the formatting of `#if`s always start at the beginning of the line. So I think it should be unindented